### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by vitacell** [Unrud/video-downloader#264]
When the program tryes to download a video, it says:

FileNotFoundError: [Errno 2] No such file or directory

**Comment by dik23** [comment]
I get the same

When I look in the download location I find a folder  which contains multiple files, depending on the source

There's always a .json a .m4a and a .mp4.  Sometimes there's a .png

The .m4a and .mp4 can be combined into a, for example, .mkv which then works as I would expect the result if the error wasn't shown

**Comment by Unrud** [comment]
URL?

**Comment by dik23** [comment]
Every URL

Examples:

https://www.bbc.co.uk/news/av/world-us-canada-68642024

https://www.reddit.com/r/AbruptChaos/comments/1bplgx1/they_saw_an_opportunity/

https://www.youtube.com/watch?v=i3svmrkl3_M

Result is always the same

I'm using the flatpak install

![image](https://github.com/Unrud/video-downloader/assets/15139568/50519cd8-0a23-4f3c-9dd6-8694fd4c7381)

**Comment by Unrud** [comment]
Run the following command to get the complete stack trace:
```shell
env G_MESSAGES_DEBUG=yt-dlp flatpak run com.github.unrud.VideoDownloader
```

**Comment by dik23** [comment]
[download.log](https://github.com/Unrud/video-downloader/files/14805165/download.log)

**Comment by dik23** [comment]
Works for me

**Comment by Unrud** [comment]
Fixed in v0.12.12. It includes a workaround for https://github.com/flatpak/xdg-desktop-portal/issues/475.
